doc/tutorials/12-constant_pH/12-constant_pH.ipynb

```diff
@@ -4,114 +4,190 @@
+    "We create a system with this box size.\n",
+    "From the salt concentration we calculate the number of additional salt ion pairs that should be present in the system.\n",
+    "We set the dissociation constant of the acid to $\\mathrm{p}K_A=4.88$, that is the acidity constant of propionic acid. We choose propionic acid because its structure is closest to the repeating unit of poly(acrylic acid), the most commonly used weak polyacid.\n",
+    "\n",
+    "We will simulate multiple pH values, the range of which is determined by the parameters `offset` and `num_pHs`."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
-    "plt.ion()\n",
     "\n",
     "import espressomd\n",
     "from espressomd import reaction_ensemble\n",
     "\n",
-    "# System parameters\n",
-    "#############################################################\n",
-    "box_l = 35 # in units of sigma\n",
+    "import scipy.constants # physical constants\n",
+    "import pint # module for working with units and dimensions\n",
+    "ureg = pint.UnitRegistry()\n",
+    "ureg.define('sigma = 0.355 * nm = sig') # sigma=0.40 nm is a commonly used  particle size in coarse-grained simulations\n",
+    "sigma = 1.0 * ureg.sigma # variable that has the value and dimension of one sigma\n",
+    "N_A=scipy.constants.N_A/ureg.mole # N_A is the numerical value of Avogadro constant in units 1/mole\n",
+    "Bjerrum=0.715 * ureg.nanometer # Bjerrum length at 300K\n",
+    "concentration = ureg.Quantity # define that concentration is a quantity that must have a value and a unit\n",
     "\n",
-    "# Integration parameters\n",
+    "# System parameters\n",
     "#############################################################\n",
-    "system = espressomd.System(box_l=[box_l] * 3)\n",
-    "system.set_random_state_PRNG()\n",
-    "np.random.seed(seed=10)\n",
-    "\n",
-    "type_HA = 0\n",
-    "type_A = 1\n",
-    "type_H = 2\n",
-    "\n",
-    "N0 = 10  # number of titratable units in the box\n",
-    "\n",
-    "for i in range(N0):\n",
-    "    system.part.add(pos=np.random.random(3) * system.box_l, type=type_HA)"
+    "c_acid=concentration(1e-3,'mol/L') # 0.05 mol/L is a reasonable concentration that could be used in experiments\n",
+    "# Using the constant-pH method is safe if Ionic_strength > max(10**(-pH), 10**(-pOH) ) and C_salt > C_acid\n",
+    "c_salt=concentration(2*c_acid) # additional salt to control the ionic strength\n",
+    "# In the ideal system, concentration is arbitrary (see Henderson-Hasselbalch equation)\n",
+    "# but it is important in the interacting system\n",
+    "N_acid = 20  # number of titratable units in the box\n",
+    "\n",
+    "PROB_REACTION=0.5 # select the reaction move with 50% probability \n",
```

Comment:
could you please check the coding style of this part? Especially:
* please use spaces in assignments: ```a = 3```
* please one assignment per line ~~```a=3; b=2```~~
* is the global system seed needed her? I thought only NPT is left using a global seed...